docs/dev/windows.md

```diff
@@ -86,9 +101,10 @@ will be performed by a single task, *i.e.* with parallelism of 1.
 
 ## Window Assigners
 
-After specifying whether your stream is keyed or not, the next step is to define a *windowing strategy*. 
-The windowing strategy defines how elements are assigned to windows. This is done by specifying the 
-`WindowAssigner` that corresponds to the windowing strategy of your choice in the `window(...)` call. 
+After specifying whether your stream is keyed or not, the next step is to define a *window assigner*. 
+The window assigner defines how elements are assigned to windows. This is done by specifying the 
+`WindowAssigner` that corresponds to the windowing strategy of your choice in the `window(...)` (for *keyed* streams)
```

Comment:
"This is done by specifying the `WindowAssigner` of your choice in the `window(...)` ...", i.e. remove "windowing strategy"